syscalls.go

```diff
@@ -173,6 +173,13 @@ type bpfProgTestRunAttr struct {
 	duration    uint32
 }
 
+type bpfProgAlterAttr struct {
+	targetFd   uint32
+	alterBpfFd uint32
```

Comment:
s/alter/attacj/g ?
https://github.com/torvalds/linux/blob/v5.3/include/uapi/linux/bpf.h#L423-L427

Reply:
as the same struct is used for attaching and detaching, I've changes it to alter. But I will revert it, to keep it the way it is in the kernel.